src/app/(proper_react)/redesign/(public)/LandingView.tsx

```diff
@@ -49,7 +49,25 @@ export const View = (props: Props) => {
         </div>
       </header>
 
-      <div className={styles.valuePropositionWrapper}>
+      <section className={styles.quoteWrapper}>
+        <div className={styles.quote}>
+          <h2>
```

Comment:
I think that maybe this isn't really a heading, since there's no content underneath it. (e.g. some tools make it easy to list all headings on a page and then to jump to a particular one, but there's not much reason to jump to this one).

Maybe the `div.quote` should just be a `<p>`, and then the `<h2>` a `<b>`?